src/main/scala/sbtorgpolicies/settings/scalafmt.scala

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2016 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sbtorgpolicies.settings
+
+import org.scalafmt.bootstrap.ScalafmtBootstrap
+import sbt.Keys._
+import sbt._
+import sbt.inc.Analysis
+
+/**
+ * https://gist.github.com/olafurpg/e045ef9d8a4273bae3e2ccf610636d66#file-automatescalafmtplugin-scala
+ */
+trait scalafmt {
+
+  val scalafmtInc: TaskKey[Unit] = taskKey[Unit]("Incrementally format modified sources")
+
+  val scalafmtGenerateFile: TaskKey[Unit] = taskKey[Unit]("Generate a default scalafmt configuration")
+
+  lazy val generateScalafmtTask = Seq(
+    scalafmtGenerateFile := Def.task {
+      if (!file(".scalafmt.conf").exists()) {
+        IO.write(
+          file(".scalafmt.conf"),
```

Comment:
Since this project is also supposed to be formatted by the same settings it enforces why don't we get just get the contents from the local `.scalafmt.conf`? :dog: :shallow_pan_of_food: